src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Backup/Blocks/backup_info.html.twig

```diff
@@ -0,0 +1,52 @@
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="alert alert-info" role="alert">
+  {% if isHostMode %}
+    <h4>{{ 'How to restore a database backup'|trans({}, 'Admin.Advparameters.Notification') }}</h4>
+    <p>{{ 'If you need to restore a database backup, we invite you to subscribe to a [1][2]technical support plan[/2][/1].'|trans({'[1]': '<strong>',  '[/1]': '</strong>', '[2]': '<a class="_blank" href="https://addons.prestashop.com/support/16298-support-essentiel-plan.html">', '[/2]': '</a>'}, 'Admin.Advparameters.Notification')|raw }}</p>
+    <p>{{ 'Our team will take care of restoring your database safely.'|trans({}, 'Admin.Advparameters.Notification') }}</p>
+    <br>
+    <p>{{ 'Why can\'t I restore it by myself?'|trans({}, 'Admin.Advparameters.Notification') }}</p>
+    <p>{{ 'Your shop is hosted by PrestaShop. Although you can create backup files here below, there are core settings you cannot access for security reasons, like the database management. Thus, only our team can proceed to a restoration.'|trans({}, 'Admin.Advparameters.Notification') }}</p>
+  {% else %}
+    <h4>{{ 'How to restore a database backup in 10 easy steps'|trans({}, 'Admin.Advparameters.Help') }}</h4>
+    <ol class="mb-0">
+      <li>{{ 'Set "Enable Shop" to "No" in the "Maintenance" page under the "Preferences" menu.'|trans({}, 'Admin.Advparameters.Help') }}</li>
+      <li>{{ 'Download the backup from the list below or from your FTP server (in the folder "admin/backups").'|trans({}, 'Admin.Advparameters.Help') }}</li>
+      <li>{{ 'Check the backup integrity: Look for errors, incomplete file, etc... Be sure to verify all of your data.'|trans({}, 'Admin.Advparameters.Help') }}</li>
+      <li>{{ 'Please ask your hosting provider for "phpMyAdmin" access to your database.'|trans({}, 'Admin.Advparameters.Help') }}</li>
+      <li>{{ 'Connect to "phpMyAdmin" and select your current database.'|trans({}, 'Admin.Advparameters.Help') }}</li>
+      <li>{{ 'Unless you enabled the "Drop existing tables" option, you must delete all tables from your current database.'|trans({}, 'Admin.Advparameters.Help') }}</li>
+      <li>{{ 'At the top of the screen, please select the "Import" tab'|trans({}, 'Admin.Advparameters.Help') }}</li>
+      <li>{{ 'Click on the "Browse" button and select the backup file from your hard drive.'|trans({}, 'Admin.Advparameters.Help') }}</li>
+      <li>
+        {{ 'Check the maximum filesize allowed (e.g. Max: 16MB)'|trans({}, 'Admin.Advparameters.Help') }} <br>
```

Comment:
Yep, this strings is currently localized in the _messages.xlf_ file but it's better with these domains!